apps/mull-ui/src/app/components/event-card/event-card.scss

```diff
@@ -14,6 +14,7 @@
 
 .event-card-image {
   object-fit: cover;
+  cursor: pointer;
```

Comment:
Instead of setting `cursor: pointer` on specific elements, it might be better to create a `.button` scss class and add it to `styles.scss`. We already have some custom styling for `button` elements found in that file, so you can apply the same styling to the `.button` class:

```scss
// in styles.scss
button, .button {
  cursor: pointer;
  outline: none;
  background-color: transparent;
}
```